### PR TITLE
Update GOVUK Flask and Jinja to latest

### PR DESCRIPTION
**Pull request by BenMillar-MOJ**
## What does this PR do?
- Pulls in the commit history from the base project, allowing us to keep a record of how this forked project compares to the original. This should make future syncs much easier.

- Bumps all requirements to the latest and matches the format used in the original project, using pip-compile.

- Increments the versions of GOVUK-WTF and GOVUK-Jinja to the latest

- Updates the project with the latest changes from the original govuk-flask-skeleton including the new GOVUK Frontend components.

- Now uses GOV.UK Frontend v.5.6.0 🎉 
- - This gives support for the new [Service Navigation component](https://design-system.service.gov.uk/components/service-navigation/).

## Things to note
- Although the original project introduced Ngnix and Redis support these were intentionally not carried over as we are not currently using these.

- Reformats the files to match the original's formatting, I am unsure when these formatting discrepancies were first introduced.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,18 +1,33 @@
+# Remove existing GOV.UK Frontend assets
```
Comment (psweeting1) [inline build.sh:1]: This is how the demo creates it assets, but we use Flask Assets to achieve this within the skeleton.

Comment (BenMillar-MOJ) [reply build.sh]: Thank you, I have removed this section with #94aa36f1

**Review by psweeting1 — APPROVED** [review]
LGTM

**Review by TawneeOwl — APPROVED** [review]
LGTM

**Pull request merged**